contrib/auto-render/index.html

```diff
@@ -9,7 +9,7 @@
     <title>Auto-render test</title>
     <script src="/katex.js" type="text/javascript"></script>
     <link href="/katex.css" rel="stylesheet" type="text/css">
-    <script src="./auto-render.js" type="text/javascript"></script>
+    <script src="/renderMathInElement.js" type="text/javascript"></script>
```

Comment:
It's be nice to get away without renaming these files as people may be relying on the existing script name.

Reply:
Agreed. Will look into it.